tasks/upload-assets-to-s3.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const mime = require('mime');
+const aws = require('aws-sdk');
+
+const defaultDirectory = 'public';
+const defaultBucket = 'ft-next-hashed-assets-prod';
+const defaultDestination = 'hashed-assets/uploaded';
+const defaultFileExtensions = ['js', 'css', 'map', 'gz', 'br'].join();
+const defaultCacheControl = 'public, max-age=31536000';
+
+function getFileType (filename) {
+	// We need to know the original file type so ignore the compression
+	const originalFile = filename.replace(/\.(br|gz)$/, '');
+	const ext = path.extname(originalFile);
+
+	return mime.getType(ext);
+}
+
+function getFileEncoding (filename) {
+	const ext = path.extname(filename);
+
+	switch (ext) {
+		case '.gz':
+			return 'gzip';
+		case '.br':
+			return 'br';
+		default:
+			return 'identity';
```

Comment:
Worth a comment along the lines of

> Indicates the identity function (i.e., no compression or modification). This token, except if explicitly specified, is always deemed acceptable

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding

Reply:
I've removed line as it's not necessary to set this